src/locales/en.json

```diff
@@ -231,6 +231,38 @@
     "aboutUsHeader": "About us",
     "aboutUsText": "We are a group of multi-national youth who found their common ground in using tech for the benefit of people. Juniper, consists of a group of 5 junior developers and 1 lead engineer working together to educate children in MENA about environmental issues and climate change through educational games. Our target group is kids living in MENA that are 5-9 years old; thus, Juniper is delivered in three main languages to help kids and their teachers/parents to easily surf the website.",
     "whyWeDoItHeader": "Why we do it",
-    "whyWeDoItText": "We have a moral obligation to preserve nature and its features, because protecting the environment protects humanity from toxic substances and our quality of food. We believe that the new generation in MENA should be supported with the right tools to believe and act for the benefit of humanitarian and social issues, on the local and global scale. Given the right tools, it is not impossible to raise a generation of activists."
+    "whyWeDoItText": "We have a moral obligation to preserve nature and its features, because protecting the environment protects humanity from toxic substances and our quality of food. We believe that the new generation in MENA should be supported with the right tools to believe and act for the benefit of humanitarian and social issues, on the local and global scale. Given the right tools, it is not impossible to raise a generation of activists.",
+    "teamInfo": {
+      "ammar": {
+        "name": "Ammar Sammour",
+        "bio": "",
+        "role": "Lead Engineer"
+      },
+      "ibrahim": {
+        "name": "Ibrahim Aljarad",
+        "bio": "Hey, I'm Ibrahim from Syria, I studied coding at Flatiron school of coding and studied economics at Alfurat university. I participated in the frontend web development bootcamp by Re-coded organization and graduated from there. My other experiences include working with several channels and agencies as a satellite producer providing the technical support for the field producing teams.",
+        "role": "Junior Developer"
+      },
+      "majd": {
+        "name": "Majd Ajroudi",
+        "bio": "A junior developer with versatile skills, experience in several engineering areas and keen interest in using tech to solve social issues.",
+        "role": "Junior Developer"
+      },
+      "sule": {
+        "name": "Sule Sinikoglu",
+        "bio": "A Web Developer who writes “good code” for creating positive impact on Global Challenges.",
+        "role": "Junior Developer"
+      },
+      "tarkan": {
+        "name": "Tarkan Mansuroglu",
+        "bio": "A “Re-Coded Front-End Web Development Bootcamp” trained and partially self-taught front-end web developer. He is also a passionate learner, PC enthusiast, gamer and fitness enthusiast. He loves solving real life problems to test and develop his skills. Follow him through social media!",
```

Comment:
use backslash escaped quotes, not the weird quotes. fix on line 253 and anywhere else too